### PR TITLE
Views

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Blog app - views
> Adding the views associated with the Models and Controllers I have already created for this Blog app

In this PR I : 
* Add views for user and post
* Create partials folder for reusable code
* Display real data from the database
* Check linters errors

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1 @@
-<h1>User Posts page</h1>
\ No newline at end of file
+<h1>User Posts page</h1>
```
Comment (DammyShittu) [inline app/views/posts/index.html.erb:1]: - [x] Kindly implement the posts index page of a user. 

  - It should render the recent comments on the posts using the defined method in the model
  - Its layout should match the wireframe.

---

  your current post index page | wireframe for post index page
  :---:|:---:
  wey | blog_user_all_posts

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,13 @@
...
+    <h3>Title : <%=@posts.title%></h3>
+    <p>Post : <%= @posts.text%> </p>
+    <p class="stats-comment"> Comments: <%= @posts.comments_counter or 0 %> | Likes: <%= @posts.likes_counter or 0 %> </p>
+  </div>
+  <div class="comment-container">
+    <% @posts.comments.each do |comment| %>
+      <p> <%= comment.user.name%>: <%= comment.text%><p>
+    <% end %>  
+  </div>
+</div>
```
Comment (DammyShittu) [inline app/views/posts/show.html.erb]: - [x] Kindly improve the layout of your post show page to match the wireframe. As you can the placements of some items are quite different from the one on the wireframe.

- [x] You also seem to be missing the post author details.

  Your Implementation | Wireframe design
  :---:|:---:
  bcg | blog_single_post

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @ChrispaixK,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (ChrispaixK) [reply app/views/posts/index.html.erb:1]: Thank you @DammyShittu , I implemented it as required, for a better understanding of what I did check the 4th user, he is the one having a post and a lot of comments

File: app/views/posts/show.html.erb (hunk 2 above)
Comment (ChrispaixK) [reply app/views/posts/show.html.erb]: Done as requested, thank you 👍🏾

**Review by julie-ify — APPROVED** [review]
Hi @ChrispaixK,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**